components/browser/session/src/main/java/mozilla/components/browser/session/SessionManager.kt

```diff
@@ -260,6 +260,19 @@ class SessionManager(
         }
     }
 
+    /**
+     * Removes a selective list of sessions.
+     */
+    fun removeSelectedSessions(sessionsToRemove: List<Session>) {
```

Comment:
nit: I think we should drop the `selected` from all the methods. This seems to be a generic "remove all those sessions" action and whether that may be from a selection or not is a detail of whoever is calling this.